packages/manager/modules/pci/src/projects/project/private-registry/components/price/price.controller.js

```diff
@@ -0,0 +1,30 @@
+import {
```

Comment:
Seems to be a duplicate of https://github.com/ovh/manager/tree/28ef2469790d31040a8bb06426da7e7fafe43f27/packages/manager/modules/enterprise-cloud-database/src/price

Is it possible to reuse the same component everywhere ?